store/aciinfo.go

```diff
@@ -21,6 +21,16 @@ import (
 	"time"
 )
 
+// ACIFetchInfo is used to store information about the fetching process of an ACI.
+type ACIFetchInfo struct {
+	// Latest defines if the ACI was imported using the latest pattern
+	// (no version label was provided on ACI discovery).
+	Latest bool
+
+	// InsecureOptions are the insecure options that was used when retrieving the ACI.
+	InsecureOptions int64
+}
```

Comment:
I agree, InsecureOptions should not be used as is in the database. For example, the flag `--insecure-options=ondisk` is not related to the image in the CAS but to the rendered tree for `rkt run`.

xref https://github.com/coreos/rkt/pull/2983#discussion_r72614147